source/kraken/fill_disruption_from_database.cpp

```diff
@@ -122,7 +122,13 @@ void fill_disruption_from_database(const std::string& connection_string,
                  "     extract(epoch from ch.updated_at  AT TIME ZONE 'UTC') :: bigint as channel_updated_at,"
                  "     cht.id as channel_type_id, cht.name as channel_type,"
                  // Property & Associate property fields
-                 "     adp.value as property_value, pr.key as property_key, pr.type as property_type"
+                 "     adp.value as property_value, pr.key as property_key, pr.type as property_type, "
+                 // Pattern and time slots
+                 "  extract(epoch from pt.start_date ) ::int as pattern_start_date, extract(epoch from pt.end_date ) "
+                 "::int as pattern_end_date, pt.weekly_pattern as pattern_weekly_pattern, "
+                 "  pt.id as pattern_id, "
+                 "  extract(epoch from ts.begin ) ::int as time_slot_begin, extract(epoch from ts.end ) as "
+                 "time_slot_end, ts.id as time_slot_id "
```

Comment:
```suggestion
                 "  extract(epoch from pt.start_date ) ::int as pattern_start_date,"
                 "  extract(epoch from pt.end_date ) ::int as pattern_end_date," 
                 "  pt.weekly_pattern as pattern_weekly_pattern, "
                 "  pt.id as pattern_id, "
                 "  extract(epoch from ts.begin ) ::int as time_slot_begin,"
                 "  extract(epoch from ts.end ) as time_slot_end," 
                 "  ts.id as time_slot_id "
```

It's already painful enough not to keep the indentation correct :D 
Nice job for having done this chance ! :+1:

Reply:
OK :+1: